baseplate/frameworks/pyramid/__init__.py

```diff
@@ -244,7 +335,6 @@ def _on_new_request(self, event: pyramid.events.ContextFound) -> None:
         span.set_tag("http.route", request.matched_route.pattern)
         span.set_tag("http.method", request.method)
         span.set_tag("peer.ipv4", request.remote_addr)
-        span.set_tag("http.request_length", request.content_length)
```

Comment:
Is there a reason to remove this?

Reply:
It waws added originally with a94f21f7d4d229757572d2f2fc714b543f9a386e for the original implementation using observers. So It's just not used is all.
Which means we should actually remove http.route too probably